mm2src/coins/hd_wallet/coin_ops.rs

```diff
@@ -231,4 +231,14 @@ pub trait HDWalletCoinOps {
 
     /// Returns the Trezor coin name for this coin.
     fn trezor_coin(&self) -> MmResult<String, TrezorCoinError>;
+
+    /// Informs the coin of the enabled address provided/derived by the hardware wallet.
+    async fn received_enabled_address_from_hw_wallet(
```

Comment:
I would prefer to have this in a separate trait in the future to start separating specific HW wallet code from the more general HD wallet code. This is not at all a priority at the current stage but we need it mentioned somewhere in our issues / checklists or at least a todo.